src/app/utility/help-panel/help-panel.component.html

```diff
@@ -16,15 +16,43 @@ <h4>Site or Source Energy?</h4>
 
 <div class="wrapper help-panel">
     <h4>Help</h4>
-    <p>Each meter's bill history can be added in the corresponding energy type tab.</p>
-    <br>
-    <p class="bold">To Do</p>
-    <ul>
-        <li>Restrict Number limits</li>
-    </ul>
+    
+    <div *ngIf="helpText === 'energy-consumption'">
+        <p>All the utility meters for <strong>{{selectedFacilityName}}</strong> can be viewed on this screen.</p>
+        <p>Click “Add New Meter” to add new utility meters.</p>
+        <p>Use “Import Data” to bulk import meter information in csv or excel format.</p>
+    </div>
+    
+    <div *ngIf="selectedSource === 'electricity' || selectedSource === 'natural-gas' || selectedSource === 'other-fuels' || selectedSource === 'other-energy' || selectedSource === 'water' || selectedSource === 'waste-water' || selectedSource === 'other-utility'">
+        <p>All the {{selectedSource}} meters for <strong>{{selectedFacilityName}}</strong> can be viewed on this screen.</p>
+        <p>Use “Add new Bill” to add monthly {{selectedSource | lowercase}} data.</p>
+        <p>Use “Import Data” to bulk import {{selectedSource | lowercase}} data in csv or excel format.</p>
+    </div>
+    
+    <p *ngIf="helpText === 'mo-meter-data'">The utility meter data has been calendarized in this screen. Use the filter menu to view desired meters and 
+        over specific period.  Data represented with <strong>N/A</strong> does not have a full month of data to calculate.</p>
+    
+    <div *ngIf="helpText === 'meter-groups'">
+        <p>The utility meters can be grouped by utility type or user preference. These groups will represent your data visualization and regression analysis.</p>
+        <p>These group can be edited by dragging the meter 
+            from one group to another. The “Ungrouped” group will disappear once you have assigned all the meters to a designated 
+            group.</p>
+    </div>
+
+    <div *ngIf="helpText === 'predictors'">
+        <p>These are independent variables that will be used for <strong>{{ selectedFacilityName }}'s</strong> regression analysis. Predictors can be used to explain the change in your utility use.</p>
+        <p>Use “Add/Edit Predictors” to Add new Predictors for your analysis.</p>
+        <p>Use “Add Predictor Entry” to input your monthly predictor data.</p>
+        <p>Use “Import Data” to bulk import Predictor data in csv or excel format.</p>
+    </div>
+    
+    <div *ngIf="helpText === 'upload-data'" class="mt-5">
+        <h4>Template Downloads</h4>
+        <ul>
```

Comment:
Relative paths don't work in electron application but we don't use these .csv templates anymore. There's just the one xlsx template now